src/Symfony/Component/PropertyAccess/Annotation/ConfigurationAnnotation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Base configuration annotation.
+ *
+ * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ */
+abstract class ConfigurationAnnotation
```

Comment:
Sorry, it's a remnant of the first approach I took. That is how annotations are implemented in the Validator component and I emulated the behaviour. Later I switched over Serializer mapping instead but forgot to modify the existing Annotation class. I'll take it out.